tests/testthat/test-add_CNSR.R

```diff
@@ -27,93 +30,108 @@ testthat::test_that("T1.1 No error when censoring is plotted for one strata with
   
   p <- adtte %>% 
     visR::estimate_KM() %>%
-    visR::plot() %>%
-    visR::add_CNSR()
-  vdiffr::expect_doppelganger("add_CNSR_T1_1", p)
+    visR::plot()
+  
+  testthat::expect_error(p %>% visR::add_CNSR(), NA)
+  
+  testthat::skip_on_cran()
```

Comment:
should the skip on cran not contain the test you want to skip eg

testthat::skip_on_cran(
  p %>% 
    visR::add_CNSR(shape = "Âµ") %>%
    vdiffr::expect_doppelganger(title = "add_CNSR_T1_5_shape_atomic_string")  
  )

Reply:
As far as I've understood the synthax, it just skips everything below the function. The function itself basically only sets an environment variable. As for example used here: https://rdrr.io/cran/ggfortify/src/tests/testthat/test-plotlib.R